test/lit.cfg

```diff
@@ -1326,6 +1326,46 @@ if platform.system() != 'Darwin':
     config.target_resilience_test = ('%s --no-symbol-diff' %
                                      config.target_resilience_test)
 
+platform_module_dir = make_path(test_resource_dir, config.target_sdk_name)
+if run_vendor != 'apple':
+    platform_module_dir = make_path(platform_module_dir, run_cpu)
+lit_config.note('Using platform module dir: ' + platform_module_dir)
+if test_sdk_overlay_dir:
+    platform_sdk_overlay_dir = test_sdk_overlay_dir
+else:
+    platform_sdk_overlay_dir = platform_module_dir
+
+# If static stdlib is present, enable static stdlib tests
+static_stdlib_path = make_path(config.swift_lib_dir, "swift_static", config.target_sdk_name)
+libswiftCore_path = make_path(static_stdlib_path, "libswiftCore.a")
```

Comment:
would be nice to sink this into the single use below to avoid this value being left around since the name is unclear whether it is static or dynamic.

Reply:
I kept it separate but changed the name to make it clear that it is static.